services/issue/assignee.go

```diff
@@ -69,5 +70,68 @@ func ReviewRequest(issue *models.Issue, doer *models.User, reviewer *models.User
 		notification.NotifyPullReviewRequest(doer, issue, reviewer, isAdd, comment)
 	}
 
+	return
+}
+
+// IsLegalReviewRequest check review request permission
+func IsLegalReviewRequest(reviewer, doer *models.User, isAdd bool, issue *models.Issue) error {
```

Comment:
I think you have to load pullRequest of the issue to remove the nil exeption @a1012112796

Reply:
and then the repo attributes of the pull ...

Reply:
but I only would load this if you like to print an error back to save time if it is not used ...

Reply:
Thanks, you are right. I forgot it.